src/CLI/actioner/sonic-cli-tacacs.py

```diff
@@ -49,20 +49,40 @@ def invoke_api(func, args):
        body = { "openconfig-system-ext:secret-key": args[0] }
        return api.patch(path, body)
     elif func == 'patch_tacacs_server':
+       indata = {}
+       # get server data
+       api_response = get_sonic_tacacs_server_api(args)
+       if api_response:
+           indata = api_response[0]
+       else:
+           # default server values
```

Comment:
So there will always be a value configured for each server's attribute, even if the user did not configure it ?

Reply:
yes: Except secret-key, there are default values for every parameter.
port:49, timeout: 5, authtype: pap, priority: 1